src/current/v23.2/molt-fetch.md

```diff
@@ -99,6 +99,44 @@ Complete the following items before using MOLT Fetch:
 
 - If a MySQL database is set as a [source](#source-and-target-databases), the [`--table-concurrency`](#global-flags) and [`--export-concurrency`](#global-flags) flags **cannot** be set above `1`. If these values are changed, MOLT Fetch returns an error. This guarantees consistency when moving data from MySQL, due to MySQL limitations. MySQL data is migrated to CockroachDB one table and shard at a time, using [`WITH CONSISTENT SNAPSHOT`](https://dev.mysql.com/doc/refman/8.0/en/commit.html) transactions.
 
+## Security
+
+Cockroach Labs **strongly** recommends the following:
+
+### Secure connections
+
+- Use secure connections to the source and target database whenever possible.
```

Comment:
I believe there are some application stacks (PHP?) where they can't use `full` but they don't need to disable. There's a spectrum. i suggest linking to https://www.cockroachlabs.com/docs/stable/connection-parameters#additional-connection-parameters.